packages/ckeditor5-image/src/imageresize/imageresizeediting.js

```diff
@@ -45,55 +37,15 @@ export default class ImageResizeEditing extends Plugin {
 		this._registerConverters();
 
 		editor.commands.add( 'imageResize', command );
-
-		editor.editing.downcastDispatcher.on( 'insert:image', ( evt, data, conversionApi ) => {
-			const widget = conversionApi.mapper.toViewElement( data.item );
-
-			const resizer = editor.plugins
-				.get( WidgetResize )
-				.attachTo( {
-					unit: editor.config.get( 'image.resizeUnit' ) || '%',
-
-					modelElement: data.item,
-					viewElement: widget,
-					editor,
-
-					getHandleHost( domWidgetElement ) {
-						return domWidgetElement.querySelector( 'img' );
-					},
-					getResizeHost( domWidgetElement ) {
-						return domWidgetElement;
-					},
-					// TODO consider other positions.
-					isCentered() {
-						const imageStyle = data.item.getAttribute( 'imageStyle' );
-
-						return !imageStyle || imageStyle == 'full' || imageStyle == 'alignCenter';
-					},
-
-					onCommit( newValue ) {
-						editor.execute( 'imageResize', { width: newValue } );
-					}
-				} );
-
-			resizer.on( 'updateSize', () => {
-				if ( !widget.hasClass( 'image_resized' ) ) {
-					editor.editing.view.change( writer => {
-						writer.addClass( 'image_resized', widget );
-					} );
-				}
-			} );
-
-			resizer.bind( 'isEnabled' ).to( command );
-		}, { priority: 'low' } );
 	}
 
 	/**
 	 * @private
 	 */
 	_registerSchema() {
-		this.editor.model.schema.extend( 'image', {
-			allowAttributes: 'width'
+		this.editor.model.schema.extend( 'image', { allowAttributes: 'width' } );
+		this.editor.model.schema.setAttributeProperties( 'width', {
```

Comment:
Was this change wasn't required by the issue? If yes - we need tests. If no we need to handle this in a separate issue to know why this change and when it is needed.

Reply:
After splitting the editing plugin I had to add this (it based on our convention to mark certain attributes as `isFormatting`). The line is covered by this test https://github.com/ckeditor/ckeditor5/blob/4fcdefe218d262441ff5a5275dae28f610ce7092/packages/ckeditor5-image/tests/imageresize/imageresizeediting.js#L121.

Reply:
I've missed that :( sorry. Let's leave it then.